protostar/testing/test_results.py

```diff
@@ -161,3 +161,12 @@ class BrokenTestSuiteResult(TestResult):
 @dataclass(frozen=True)
 class UnexpectedBrokenTestSuiteResult(BrokenTestSuiteResult):
     traceback: Optional[str]
+
+
+NONNEGATIVE_RESULTS = (
```

Comment:
Why do you use a caps-locked name for this? Also, I'd include "test", maybe something like this: `NonnegativeTestResults`?